add Karen Word's Trainer Training blog post (#606)
carpentries/carpentries.org: content/posts/2026/02/end-of-support-for-styles.md

---  
layout: page  
authors: ["Toby Hodges"]  
teaser: "Bidding a fond farewell to our previous lesson infrastructure."  

> title: "End of Support for *styles*"  

Reviewer: ```suggestion
teaser: "Bidding a fond farewell to the former lesson template, _styles_."  
title: "End of Support for Our Previous Lesson Infrastructure"  
```

Since Markdown stying doesn't work on the post title, rewording to put _styles_ in the teaser text instead.

Reviewer 2: Thanks Toby! My previous commits were also trying to fix the same errors you picked up!

Over the last twelve months, more lesson developers either transitioned their repositories to the Workbench, following our migration workflow, or requested that the Curriculum Team make the transition for them.

## What went well?

I am delighted with the results: at the time of writing, The Carpentries Incubator includes 136 lesson repositories that are using the Workbench infrastructure, with transition pending on 14 more repositories. 63 lesson repositories have been archived, many of which were dormant or had received no commits since their initial creation. The Curriculum Team has transitioned 15 Incubator and Lab repositories to the Workbench since 2025, and a handful more where we received requests for help from projects outside of The Carpentries.

> ![A diagram indicating the transition status of all 213 lesson repositories in the Incubator](/blog/2026/02/subsetting-status-incubator.png)

Reviewer: ```suggestion
![A diagram indicating the transition status of all 213 lesson repositories in the Incubator](/blog/2026/02/sunsetting-status-incubator.png)
```

---  
layout: page  
authors: ["Toby Hodges"]  

> teaser: "Bidding a fond farewell to the former lesson template, _styles_."  

Reviewer: ```suggestion
teaser: "Bidding a fond farewell to the former lesson template."  
```

## What went well?

I am delighted with the results: at the time of writing, The Carpentries Incubator includes 136 lesson repositories that are using the Workbench infrastructure, with transition pending on 14 more repositories. 63 lesson repositories have been archived, many of which were dormant or had received no commits since their initial creation. The Curriculum Team has transitioned 15 Incubator and Lab repositories to the Workbench since 2025, and a handful more where we received requests for help from projects outside of The Carpentries.

![A diagram indicating the transition status of all 213 lesson repositories in the Incubator](/blog/2026/02/sunsetting-status-incubator.png)

## What was difficult?

The transition of an existing lesson from *styles* to the Workbench is a complex and delicate process. Community members reported that the documented workflow was difficult to follow and too sensitive to differences in the local computing environment. These factors created a greater dependency on the Curriculum Team to support lesson transitions.

> Thankfully, once successfully configured, the lesson transition process was usually very fast. The Curriculum Team developed plenty of experience with the kinds of formatting issues that could disrupt a transition and were able to successfully migrate the community's lessons relatively quickly. Nevertheless, weâ€™re grateful to be able to move on to the next project!

Reviewer 3: ```suggestion
Thankfully, once successfully configured, the lesson transition process was usually very fast. The Curriculum Team developed plenty of experience with the kinds of formatting issues that could disrupt a transition and were able to successfully migrate the community's lessons relatively quickly. Nevertheless, we're grateful to be able to move on to the next project!
```

## What difference does this make?

> Consistency of infrastructure used across all of the lesson repositories in our ecosystem has many advantages. It gives us greater confidence that we can help lesson developers with troubleshooting, ensures that all lessons benefit from the increased accessibility provided by the Workbench, and further simplifies the process of providing further infrastructure updates.

Reviewer 3: ```suggestion
Consistency of infrastructure used across all of the lesson repositories in our ecosystem has many advantages. It gives us greater confidence that we can help lesson developers with troubleshooting, ensures that all lessons benefit from the increased accessibility provided by the Workbench, and further simplifies the process of providing future infrastructure updates.
```

Reviewer 3: Suggestion to avoid duplicate use of "further".

> Thank you to everyone who helped make this process a success, and to the developers and Maintainers of the *styles* infrastructure that this community built so many fantastic lessons with over the years.

Reviewer 3: ```suggestion
Thank you to everyone who helped make this process a success, to the developers and Maintainers of the *styles* infrastructure that this community built so many fantastic lessons with over the years, and to the developers and contributors to the Workbench who are carrying this work into the future!
```

Reviewer 3: I wanted to make sure we also thank folks who have contributed to the Workbench. Please feel free to change the language.
